packages/arb-token-bridge-ui/src/state/cctpState.ts

```diff
@@ -190,7 +184,7 @@ export const useCCTPWithdrawals = ({
   pageSize,
   enabled
 }: fetchCctpParams) => {
-  const { data, error, isLoading, isValidating } = useSWRImmutable(
+  const { data, error, isLoading, mutate } = useSWRImmutable(
```

Comment:
can we just `return useSWRImmutable` here instead of destructuring the response? We can simply use the required keys from it wherever we want, then.

Reply:
> can we just return useSWRImmutable here instead of destructuring the response?

We can.

> We can simply use the required keys from it wherever we want, then.

I'm not sure to understand what you mean by this? Ideally, this should never be changed directly from outside of this hook